src/open_dread_rando/files/levels/s010_cave.lc.lua

```diff
@@ -143,6 +143,8 @@ function s010_cave.InitFromBlackboard()
   if not CAVES_GAME_INTRO then
     s010_cave.OnEnd_Cutscene_intro_end()
   end
+
+  Game.AddSF(1.0, "Scenario.Rando_SetAreaLocations", "i", 35) -- EMMI-02SM and Corpius
```

Comment:
instead of doing it like this, put it directly in Scenario.InitFromBlackboard. something like:
```lua
Scenario.tNumTanksMaxByScenario = {
    "s010_cave" = 35,
    -- etc.
}

function Scenario.InitFromBlackboard()
    -- ...
    Game.AddSF(1.0, "Scenario.Rando_SetAreaLocations", "i", Scenario.tNumTanksMaxByScenario[CurrentScenarioID])
end
```